firefox_desktop/views/desktop_daily_retention.view.lkml

```diff
@@ -0,0 +1,355 @@
+
+  # Hide metric columns showing as dimensions
+  dimension: cohort {
+    hidden: yes
+    sql: ${TABLE}.cohort ;;
+  }
+
+  dimension: num_clients_active_atleastonce_in_last_28_days {
+    hidden: yes
+    sql: ${TABLE}.num_clients_active_atleastonce_in_last_28_days ;;
+  }
+
+  dimension: num_clients_active_atleastonce_in_last_7_days {
+    hidden: yes
+    sql: ${TABLE}.num_clients_active_atleastonce_in_last_7_days ;;
+  }
+
+  dimension: num_clients_active_on_day {
+    hidden: yes
+    sql: ${TABLE}.num_clients_active_on_day ;;
+  }
+
+  dimension: num_clients_dau_active_atleastonce_in_last_28_days {
+    hidden: yes
+    sql: ${TABLE}.num_clients_dau_active_atleastonce_in_last_28_days ;;
+  }
+
+  dimension: num_clients_dau_active_atleastonce_in_last_7_days {
+    hidden: yes
+    sql: ${TABLE}.num_clients_dau_active_atleastonce_in_last_7_days ;;
+  }
+
+  dimension: num_clients_dau_on_day {
+    hidden: yes
+    sql: ${TABLE}.num_clients_dau_on_day ;;
+  }
+
+  dimension: num_clients_dau_repeat_first_month_users {
+    hidden: yes
+    sql: ${TABLE}.num_clients_dau_repeat_first_month_users ;;
+  }
+
+  dimension: num_clients_in_cohort {
+    hidden: yes
+    sql: ${TABLE}.num_clients_in_cohort ;;
+  }
+
+  dimension: num_clients_repeat_first_month_users {
+    hidden: yes
+    sql: ${TABLE}.num_clients_repeat_first_month_users ;;
+  }
+
+  # Custom
+
+  # Define measures
+  measure: cohort_measure {
+    label: "New Profiles (cohort)"
+    type:  sum
+    sql: (${TABLE}.num_clients_in_cohort) ;;
+  }
+
+  measure: num_clients_active_on_day_measure {
+    label: "clients - sent ping on day"
+    type: sum
+    sql:  ${TABLE}.num_clients_active_on_day ;;
+    hidden: yes
+  }
+
+  measure: num_clients_dau_on_day_measure {
+    label: "clients - qualified as DAU on day"
+    type: sum
+    sql:  ${TABLE}.num_clients_dau_on_day ;;
+    hidden: yes
+  }
+
+  measure: num_clients_active_atleastonce_in_last_7_days_measure {
+    label: "clients - sent ping at least once in last 7 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_active_atleastonce_in_last_7_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_dau_active_atleastonce_in_last_7_days_measure {
+    label: "clients - DAU at least once in last 7 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_dau_active_atleastonce_in_last_7_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_active_atleastonce_in_last_28_days_measure {
+    label: "clients - sent ping at least once in last 28 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_active_atleastonce_in_last_28_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_dau_active_atleastonce_in_last_28_days_measure {
+    label: "clients - DAU at least once in last 28 days"
+    type: sum
+    sql:  ${TABLE}.num_clients_dau_active_atleastonce_in_last_28_days ;;
+    hidden: yes
+  }
+
+  measure: num_clients_active_in_period {
+    label: "clients - sent a ping during the retention period "
```

Comment:
This is very confusing to me. DAU is `Daily Active Users`, yet this is different than our DAU measure.

In reality, I'd hide this field entirely to reduce confusion. We definitely don't want users to have to think in terms of "pings" for these kinds of analyses.

Reply:
I ended up hiding the non-dau version.